primitiveFTPd/src/org/primftpd/prefs/PrefsBean.java

```diff
@@ -26,6 +26,7 @@
 	private final StorageType storageType;
 	private final String safUrl;
 	private final String allowedIpsPattern;
+	private final boolean quickSettingsRequiresUnlock;
 
```

Comment:
It is not necessary to have new preference in PrefsBean as it is used in QuickSettingsService only. And There it is loaded by static method, without using PrefsBean